app/javascript/modules/listingDetailsAside/ListingDetailsWaitlist.tsx

```diff
@@ -17,7 +17,13 @@ export const ListingDetailsWaitlist = ({ listing }: ListingDetailsWaitlistProps)
     </>
   )
 
-  const waitlistAvailableDescription = <p>{t("listings.availableUnitsAndWaitlistDescription")}</p>
+  const waitlistAvailableDescription = (
+    <p>
+      {t("listings.availableUnitsAndWaitlistDescription", {
+        number: listing.Total_number_of_building_units,
```

Comment:
Would be a #small-win if we wanted to support single vs plural - just check if the `listing.Total_number_of_building_units` is 1 or greater and use the appropriate text prop. That being said, we've spent plenty of time talking about the waitlist component. I'm ok with carrying forward the existing functionality.

Reply:
sounds good! I didn't do it because all the different translations needed for that felt like it went beyond the scope